browser/js/kit/kit.controllers.js

```diff
@@ -1,8 +1,23 @@
-app.controller('KitsController', function ($scope, allKits, allKitImages) {
-	
+app.controller('KitsController', function ($scope, allKits, allKitImages, AuthService, $uibModal, $log) {
+	AuthService.getLoggedInUser().then(function(user){
+		$scope.user= user;
+	})
 	$scope.kits = allKits
 	$scope.images = allKitImages
 	$scope.title= 'All Kits'
+	$scope.open = function (size) {
```

Comment:
@jdhang I think the "add a kit" button is hidden for non-users, and the button has an ng-show if user is an admin.  The function $scope.open is only available on the button (which is hidden for non-users). The function itself does not check admin status - only the button does.